tools/codegen/gofilegen/schema/schema_attribute.go

```diff
@@ -105,49 +105,54 @@ func generator(attr *codespec.Attribute) attributeGenerator {
 }
 
 // generation of conventional attribute types which have common properties like MarkdownDescription, Computed/Optional/Required, Sensitive
-func commonAttrStructure(attr *codespec.Attribute, typeDef string, specificProperties []CodeStatement) CodeStatement {
-	properties := commonProperties(attr)
-	imports := []string{}
-
-	if attr.CustomType != nil {
-		imports = append(imports, "github.com/hashicorp/terraform-plugin-framework-jsontypes/jsontypes")
-	}
-
-	for i := range specificProperties {
-		properties = append(properties, specificProperties[i].Code)
-		imports = append(imports, specificProperties[i].Imports...)
-	}
+func commonAttrStructure(attr *codespec.Attribute, attrDefType, planModifierType string, specificProperties []CodeStatement) CodeStatement {
+	properties := commonProperties(attr, planModifierType)
+	properties = append(properties, specificProperties...)
 
 	name := attr.TFSchemaName
-	propsResultString := strings.Join(properties, ",\n") + ","
+	result := GroupCodeStatements(properties, func(properties []string) string {
```

Comment:
Would rename `result` to `propsStmts` or similar.
```suggestion
	propsStmts := GroupCodeStatements(properties, func(properties []string) string {
```